lib/util/package-rules/managers.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { ManagersMatcher } from './managers';
+
+describe('util/package-rules/managers', () => {
+  const managersMatcher = new ManagersMatcher();
+
+  describe('match', () => {
+    it('should match againt custom manager when macthManager=custom', () => {
```

Comment:
```suggestion
    it('should match againt custom manager when matchManager=custom', () => {
```

Reply:
```suggestion
    it('should match against custom manager when matchManager=custom', () => {
```

Actually, there are two typos. 😉